app/controllers/index.js

```diff
@@ -1,7 +1,74 @@
 import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
```

Comment:
FYI, in Ember 4.1+ a new import is available which is shorter:

```suggestion
import { service } from '@ember/service';
```